executor/aggregate.go

```diff
@@ -798,13 +800,15 @@ func (e *HashAggExec) execute(ctx context.Context) (err error) {
 
 func (e *HashAggExec) getPartialResults(groupKey string) []aggfuncs.PartialResult {
 	partialResults, ok := e.partialResultMap[groupKey]
+	mSize := int64(unsafe.Sizeof(e.partialResultMap))
```

Comment:
Hi, `unsafe.Sizeof` can only trace the memory usage of a go map itself, but what is more important is that tracing the value of the map, which is the memory usage of `aggfuncs.PartialResult`.